src/main/java/com/school/project/service/impl/GroupServiceImpl.java

```diff
@@ -16,13 +19,19 @@
     @Autowired
     GroupRepository groupRepository;
 
+    @Autowired
+    UserRepository userRepository;
```

Comment:
do you use it somewhere? can't find any usage

Reply:
yes, i use it here:
 @Override
    public Group addUser(Long groupId, Long userId) {
        User userById = userService.getUserById(userId);  (line 49)